packages/core/src/main/test/CryptographyHelper.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {Cryptobox} from '@wireapp/cryptobox';
+import * as Proteus from '@wireapp/proteus';
+import {PreKey} from '@wireapp/proteus/dist/keys';
+import * as bazinga64 from 'bazinga64';
+import {CryptographyService} from '../cryptography';
+
+const StoreHelper = require('./StoreHelper');
```

Comment:
Why not rewrite the StoreHelper, too?

Reply:
Because it's currently used in some JS code (browser tests).